main.go

```diff
@@ -118,24 +117,25 @@ func main() {
 		setupLog.Error(err, "Failed to get 'NAMESPACE' environment variable")
 		os.Exit(1)
 	}
-
+	availableCRDs := mapCRDAvailability()
```

Comment:
Add space as this var is going to be used in multiple sections below,
```suggestion
	availableCRDs := mapCRDAvailability()
	
```